pkg/machines/components/vm/dummyVm.jsx

```diff
@@ -59,4 +59,6 @@ DummyVm.propTypes = {
     vm: PropTypes.object.isRequired,
 };
 
-export default DummyVm;
+export function dummyVmsConvert(vms, uiVms) {
+    return uiVms.filter(uiVm => vms.find(vm => vm.name == uiVm.name && vm.connectionName == uiVm.connectionName) == undefined);
```

Comment:
`== undefined` is too fuzzy, e. g. `null == undefined` is true in JS. Please use `===` for comparisons against null or undefined.